tests/tasks/setup/eks/awscli-cfn-lt-al2023.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cfn-launch-template-al2023
```

Comment:
Can we merge this  task with already existing [task](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/tasks/setup/eks/awscli-cfn-lt.yaml) ?  



Majority of the stuff is a repeat like [this](https://github.com/awslabs/kubernetes-iteration-toolkit/pull/488/files#diff-09a9b23e589ffd628197632a0b67671b48cc00b38f4ebad7b95e65e5a8b6e274R47-R68) and [this](https://github.com/awslabs/kubernetes-iteration-toolkit/pull/488/files#diff-09a9b23e589ffd628197632a0b67671b48cc00b38f4ebad7b95e65e5a8b6e274R85-R96 )  ?


wdyt ?